packages/fxa-settings/src/pages/InlineRecoverySetupFlow/container.tsx

```diff
@@ -73,6 +74,16 @@ export const InlineRecoverySetupFlowContainer = ({
   const location = useLocation() as ReturnType<typeof useLocation> & {
     state?: SigninRecoveryLocationState;
   };
+  // For the inline recovery flow during a multi-account sign-in, target GQL requests
```

Comment:
Reading this later, "multi-account sign-in" sounds a little confusing, can we say something similar to your note in the InlineRecoverySetup container?

Also... I could totally be missing something here, but why do we need this `gqlAuthContext` bit? Can we just set the fetch policy to network-only? I quickly tried it [here with this gist](https://gist.github.com/LZoog/c70f95b55077e92c3e30fe07800f2993), and I'm not reproducing the bug. I'll keep testing though.

Reply:
Totally right, changing the fetch policy looks to be sufficient. I've updated and removed the cruft